src/main/modules/nunjucks/index.ts

```diff
@@ -44,7 +44,7 @@ export class Nunjucks {
         classes: i.classes,
         value: i.value ?? userAnswer?.[i.name as string] ?? (userAnswer as string),
         attributes: i.attributes,
-        checked: i.selected ?? userAnswer?.[i.name as string] ?? i.value === userAnswer,
+        checked: i.selected || i.value === userAnswer?.[i.name as string] || i.value === userAnswer,
```

Comment:
in multiple checkboxes situation, i use this userAnswer?.[i.name as string]?.includes(i.value as string) as the answers will be an array. will this work?

Reply:
👍 I've updated this to match #446